pytest_tipsi_django/plugin.py

```diff
@@ -75,14 +76,15 @@ def auto_transaction(request):
 
 @pytest.fixture(scope='session', autouse=True)  # noqa
 def local_cache(session_settings, django_db_setup):
-    session_settings.DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
-    session_settings.BROKER_BACKEND = 'memory'
-    session_settings.CACHES = {
-        'default': {
+    if not getattr(session_settings, 'DEFAULT_FILE_STORAGE', None):
+        session_settings.DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
```

Comment:
We had an agreement only about CACHES, so DEFAULT_FILE_STORAGE and BROKER_BACKEND should remain untouched